palindrome-products/palindrome-products_test.spec.js

```diff
@@ -7,7 +7,7 @@ describe("Palindrome", function() {
     var largest = palindromes.largest;
 
     expect(largest.value).toEqual(9);
-    expect([1, 9]).toEqual(largest.factors);
+    expect([[3, 3], [1, 9]], [[1, 9], [3, 3]]).toContain(largest.factors);
   });
 
   xit("largets palindrome from double digit factors", function() {
```

Comment:
@rchavarria -- weird, I did fix the typo.  I must have forgotten to push the commit.  

Are you saying the test I committed is incorrect?  Sorry to be dense...